common/lib/dependabot/file_fetchers/base.rb

```diff
@@ -861,18 +858,40 @@ def decode_binary_string(str)
 
       sig { params(path: String).returns(T::Array[String]) }
       def find_submodules(path)
-        SharedHelpers.run_shell_command(
-          <<~CMD
-            git -C #{path} ls-files --stage
-          CMD
-        ).split("\n").filter_map do |line|
+        lfs_enabled = is_lfs_enabled(path) if lfs_enabled.nil?
```

Comment:
```suggestion
        lfs_enabled = is_lfs_enabled(path) if lfs_enabled.nil?
```

Looks like we're calling `.nil?` on `lfs_enabled` before it is defined, what exactly are we trying to do here?